converters/fromZigbee.js

```diff
@@ -145,6 +145,28 @@ const converters = {
             return result;
         },
     },
+    light_color_options: {
```

Comment:
This can be merged with: https://github.com/Koenkk/zigbee-herdsman-converters/blob/759c288a5468a8fd8cf4f7c67994661d4799299a/converters/fromZigbee.js#L502